services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -163,26 +164,27 @@ export const RateDetails = ({
             data?.indexHealthPlanPackages.edges
                 .map((edge) => edge.node)
                 .forEach((sub) => {
-                    const currentRevision = sub.revisions[0]
```

Comment:
We could sort by date of submitted and draft submissions after `.map((edge) => edge.node)`.